src/routes/paths.js

```diff
@@ -64,6 +64,7 @@ export const PATH_APP = {
     root: path(ROOTS.app, '/management'),
     user: {
       root: path(ROOTS.app, '/management/user'),
+      logEntries: path(ROOTS.app, '/management/user/log-entries'),
```

Comment:
I also think we shouldn't have log-entries under /user

The idea is to see today's log entries for a given child, so I imagine the root being something like this `/children/{id}/log-entries`. We can discuss this tomorrow.